src/Compilers/CSharp/Test/Syntax/Parsing/MemberDeclarationParsingTests.cs

```diff
@@ -1870,6 +1870,373 @@ public void RequiredModifierIncompleteMember_05()
             EOF();
         }
 
+        [Fact]
+        public void TypeNamedRequired_CSharp10()
```

Comment:
There's no type named required in this test, unlike the file version :).

Reply:
I'll add back 'class required' here, perhaps it's helpful to be clear about how it parses in each language version.